### PR TITLE
Rankingnew

### PR DESCRIPTION
**Pull request by Yongyao**
Initial ranking algorithm implemented
Comment out autocompletion for now

### CODE REVIEW

File: bin/mudrod [hunk 1]
```diff
@@ -1,80 +0,0 @@
-#!/bin/bash
```
Comment (lewismc) [inline bin/mudrod:1]: Why are you deleting this code?

File: core/src/main/java/esiptestbed/mudrod/ssearch/ClickstreamImporter.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
+package esiptestbed.mudrod.ssearch;
```
Comment (lewismc) [inline core/src/main/java/esiptestbed/mudrod/ssearch/ClickstreamImporter.java:1]: Add license header please.

File: core/src/main/java/esiptestbed/mudrod/ssearch/ClickstreamImporter.java [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+import esiptestbed.mudrod.discoveryengine.MudrodAbstract;
+import esiptestbed.mudrod.driver.ESDriver;
+import esiptestbed.mudrod.driver.SparkDriver;
+
+public class ClickstreamImporter extends MudrodAbstract {
+
+  public ClickstreamImporter(Map<String, String> config, ESDriver es, SparkDriver spark) {
+    super(config, es, spark);
+    // TODO Auto-generated constructor stub
```
Comment (lewismc) [inline core/src/main/java/esiptestbed/mudrod/ssearch/ClickstreamImporter.java:23]: Remove useless comment.

File: core/src/main/java/esiptestbed/mudrod/ssearch/Searcher.java [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+  private static final Logger LOG = LoggerFactory.getLogger(ESDriver.class);
+  DecimalFormat NDForm = new DecimalFormat("#.##");
+  /**
+   * @param config  Hashmap read from config.xml
+   * @param es      Elasticsearch client
+   * @param spark   Spark connection
+   */
+  public Searcher(Map<String, String> config, ESDriver es, SparkDriver spark) {
+    super(config, es, spark);
+    // TODO Auto-generated constructor stub
```
Comment (lewismc) [inline core/src/main/java/esiptestbed/mudrod/ssearch/Searcher.java:58]: Remove useless comment.

**Comment by lewismc** [comment]
I've made some code level comments. It would be also useful for you to add in Class level Javadoc for the time being. It means that classes are documented and available for use as developer docs.

File: bin/mudrod (hunk 1 above)
Comment (Yongyao) [inline bin/mudrod:1]: I did this unintentionally. What is the use of this file?

**Comment by Yongyao** [comment]
Thanks, I will revise it and submit the revision together with another branch. For now, I will just close this one.

**Pull request closed without merging**